scylla/src/transport/retry_policy.rs

```diff
@@ -374,13 +374,13 @@ mod tests {
     // On ReadTimeout we retry one time if there were enough responses and the data was present no matter the idempotence
     #[test]
     fn default_read_timeout() {
-        // Enough responses and data_present == true
+        // Enough responses and data_present == false - coordinator received only checksums
         let enough_responses_with_data = QueryError::DbError(
             DbError::ReadTimeout {
                 consistency: LegacyConsistency::Regular(Consistency::Two),
                 received: 2,
                 required: 2,
-                data_present: true,
+                data_present: false,
```

Comment:
Could you also swap the names?
Now `_with_data` doesn't have data present and `no_data` has data present.